llm/deepseek-r1/README.md

```diff
@@ -1,37 +1,36 @@
-# Distributed DeepSeek-R1 Serving with high throughput using SGLang and SkyPilot
-
+# Distributed DeepSeek-R1 Serving with High Throughput using SGLang and SkyPilot
 
 <p align="center">
 <img src="https://i.imgur.com/cd5C5IK.png" alt="DeepSeek-R1 on SkyPilot" style="width: 70%;">
 </p>
 
 On Jan 20, 2025, DeepSeek AI released the [DeepSeek-R1](https://github.com/deepseek-ai/DeepSeek-R1), including a family of models up to 671B parameters. 
 
-DeepSeek-R1 naturally emerged with numerous powerful and interesting reasoning behaviors. It outperforms **state-of-the-art proprietary models** such as OpenAI-o1-mini and becomes **the first time** an open LLM closely rivals like OpenAI-o1.
+DeepSeek-R1 naturally emerged with numerous powerful and interesting reasoning behaviors. It outperforms **state-of-the-art proprietary models** such as OpenAI-o1-mini and becomes **the first open LLM** to closely rival closed-source models like OpenAI-o1.
 
 We use [SGLang](https://github.com/sgl-project/sglang) to serve the model distributedly with high throughput in this example.
 
 
-**Note**: This example is for the original DeepSeek-R1 671B model. For smaller size distilled models, please refer to [deepseek-r1-distilled](https://github.com/skypilot-org/skypilot/tree/master/llm/deepseek-r1-distilled/).
+**Note**: This example is for the original DeepSeek-R1 671B model. For smaller distilled models, please refer to [deepseek-r1-distilled](https://github.com/skypilot-org/skypilot/tree/master/llm/deepseek-r1-distilled/).
 
 ## Run 671B DeepSeek-R1 on Kubernetes or any Cloud
 
-SkyPilot allows you to run the model distributedly with a single command with the framework [SGLang](https://github.com/sgl-project/sglang).
```

Comment:
Would rather keep this.

Reply:
Oops, my mistake. Fixed the typo!

Reply:
I see your point! The double "with" feels a bit redundant to me. Would this sound better to you?

> SkyPilot allows you to run the model distributedly with a single command, leveraging the framework [SGLang](https://github.com/sgl-project/sglang).